compiler/src/iree/compiler/Codegen/Dialect/LoweringConfig.td

```diff
@@ -104,18 +104,22 @@ def IREECodegen_TranslationInfoAttr :
   }];
 
   let assemblyFormat = [{
-    `<` `` $passPipeline (`,` `workload_per_wg` `=` $workloadPerWorkgroup^)? `>`
+    `<` `` $passPipeline (`,` `workload_per_wg` `=` $workloadPerWorkgroup^)?
+    (`pipeline_depth` `=` $softwarePipelineDepth^)? `>`
```

Comment:
s/`(`pipeline_depth` `=` $softwarePipelineDepth^)?`/`(`,` `pipeline_depth` `=` $softwarePipelineDepth^)?`/

Reply:
Yes I tried this, but then the parser complains that workload_per_wg is not specified, any ideas how to not have that issue?